_data/hosting_providers.json

```
@@ -1929,6 +1929,16 @@
     "reviewed": "2019.6.6",
     "note": ""
   },
+  {
+    "name": "Vimexx",
+    "link": "",
```

`link` is a required field. I didn't notice it was missing until I tried to view the change locally, really sorry!

```suggestion
    "link": "https://www.vimexx.nl/",
```

---

Hm, curious, as most of the entries lack a link. I didn't see the relevance of such a link anyway, therefore I omitted it.

---

JSON doesn't support comments, no. We link to [these instructions](https://github.com/certbot/website#hosting-providers-list) from the website itself, but I guess easy to miss.

I think the entries lacking a link were brought over from some old source, which didn't have any links.